src/web_interface/file_tree/file_tree.py

```diff
@@ -120,15 +76,21 @@ def root_is_virtual(root: List[dict]) -> bool:
 
 def remove_virtual_path_from_root(root: List[dict]) -> List[dict]:
     '''
-    when a file object is the root, the directories that contain the file object need to be removed so that the file
-    tree is displayed correctly in the web interface
+    When a file object is the root, the directories that contain the file object need to be removed so that the file
+    tree is displayed correctly in the web interface.
```

Comment:
Also in other instances: No `:param:` definition. Purposely omitted or forgotten?

Reply:
added parameter docs